sdk/servicebus/service-bus/test/serviceBusClient.spec.ts

```diff
@@ -282,6 +282,7 @@ describe("Errors with non existing Queue/Topic/Subscription", async function():
   it("throws error when receving streaming data from a non existing queue #RunInBrowser", async function(): Promise<
     void
   > {
+    errorWasThrown = false;
```

Comment:
The `beforeEach()` section at https://github.com/Azure/azure-sdk-for-js/blob/09dda6f24806aec08dfb90df3ac889af9e370da4/sdk/servicebus/service-bus/test/serviceBusClient.spec.ts#L195-L198 does set `errorWasThrown` to `false`.